README.md

```diff
@@ -10,31 +10,68 @@ With the public release of `$vectorSearch`, we have needed to integrate into the
 ([LangChain](https://github.com/langchain-ai/langchainjs), [LlamaIndex](https://github.com/run-llama/llama_index), [Semantic Kernel](https://github.com/microsoft/semantic-kernel)... etc) This repository runs continuous testing against each of these repos.
 
 ## How to add a test
+> **NOTE** All tests run against this repo are now required to work against a local Atlas deployment. See details below to ensure proper setup.
 
 ### Test Layout
 
 Each AI/ML pipeline is sorted by the composite of the name of the library, and the driver language the library is implemented in. This comes out in the format `{pipeline}-{language}` --> `semantic-kernel-python`. All tests should be scoped within the bounds of these subdirectories.
 
-Within each subdirectory you should expect to have a `run.sh` file. This run script will handle any additional installation, or scaffolding steps to get the test suite running. Note, it does not need to include steps to scaffold the atlas database. Additionally, you can add useful environment files such as a `.env` file or atlas vector search index configurations.
+Within each subdirectory you should expect to have:
+- `run.sh`  --  A script that should handle any additional library installations and steps for executing the test suite. This script should not populate the Atlas database with any required test data.
+- `database/` -- An optional directory used by `.evergreen/scaffold_atlas.py` to populate a MongoDB database with test data. Only provide this if your tests require pre-populated data.
+- `database/{collection}.json` -- An optional JSON file containing one or more MongoDB documents that will be uploaded to `$DATABASE.{collection}` in the local Atlas instance. Only provide this if your tests require pre-populated data.
+- `indexConfig.json` -- An optional file containing configuration for a specified Atlas Search Index.
+- Additionally, you can add other useful files, including `.env` files, if required by your tests.
 
 Each subdirectory holds the space for one repository, and within each subdirectory is where evergreen will clone the specified third-party library.
 
-### Unpacking the Evergreen
+The general layout of this repo will looks like this:
+```bash
+├── LICENSE										# License Agreeement
+├── README.md									# This Document
+├── langchain-python							# Folder scoped for one Integration
+│   └── run.sh									# script that executes test
+├── semantic-kernel-csharp						# Folder scoped for one Integration
+│   ├── database								# Optional database definition
+│   │   └── nearestSearch.json					# Populates $DATABASE.nearestSearch
+│   │   └── furthestSearch.json					# Populates $DATABASE.furthestSearch
+│   ├── indexConfig.json						# Creates Search Index on $DATABASE
+│   └── run.sh									# script that executes test
+```
```

Comment:
```suggestion
The general layout of this repo will looks like this:
```bash
├── LICENSE										# License Agreeement
├── README.md									# This Document
├── langchain-python							# Folder scoped for one Integration
│   └── run.sh									# script that executes test
├── semantic-kernel-csharp						# Folder scoped for one Integration
│   ├── database								# Optional database definition
│   │   └── nearestSearch.json					# Populates $DATABASE.nearestSearch
│   │   └── furthestSearch.json					# Populates $DATABASE.furthestSearch
│   ├── indexConfig.json						# Creates Search Index on $DATABASE
│   └── run.sh									# script that executes test
├── semantic-kernel-python						# Folder scoped for one Integration
│   ├── database								# Optional database definition
│   │   └── nearestSearch.json					# Populates $DATABASE.nearestSearch
│   │   └── furthestSearch.json					# Populates $DATABASE.furthestSearch
│   ├── indexConfig.json						# Creates Search Index on $DATABASE
│   └── run.sh									# script that executes test
```
```